src/main/config/config.c

```diff
@@ -591,7 +591,11 @@ static void resetConf(void)
 #endif
 
     // Radio
+#ifdef RX_CHANNELS_TAER
```

Comment:
You can do this as part of this one? I don't mind. Otherwise I need to wait until this is merged.